variables.tf

```diff
@@ -457,3 +457,9 @@ variable "vnet_subnet_id" {
   description = "(Optional) The ID of a Subnet where the Kubernetes Node Pool should exist. Changing this forces a new resource to be created."
   default     = null
 }
+
+variable "ultra_ssd_enabled" {
```

Comment:
Could we move this block to line 455 please? Just below `var.tags`. We're sorting variables by the following order:

1. Required variables(without default value) in alphabet order
2. Optional variables in alphabet order

Thanks for your understanding!